proxy/spanner/src/main/resources/application.conf

```diff
@@ -19,6 +19,24 @@ akka {
     }
 
     spanner {
+      journal {
+        # name of the table to use for events
+        table = "journal"
+        table = "journal_"${?SPANNER_TABLE_PREFIX}
+
+        # name of the (interleaved) table used for tags
+        event-tag-table = "tags"
+        event-tag-table = "tags_"${?SPANNER_TABLE_PREFIX}
+
+        # name of the auxilary table used to store information about deletes
+        deletions-table = "deletions"
+        deletions-table = "deletions_"${?SPANNER_TABLE_PREFIX}
+      }
+
+      snapshot {
+        table = "snapshots"
+        table = "snapshots_"${?SPANNER_TABLE_PREFIX}
+      }
```

Comment:
All these table names are also defined further down the file under `cloudstate.proxy.spanner`. Maybe we should have the table names only defined once, and use setting references?

Reply:
That makes sense to me.

Reply:
Also, these tables will be called `journal_`, `tags_`, and so on, when `SPANNER_TABLE_PREFIX` is not set. The initial setting will always be overridden given the string prefix (different from if it was just `table = ${?TABLE}`. I don't know if that matters (maybe table names can't end with an underscore?) if the prefix var is always set — although could then be better to make it required.

Reply:
Lets loop @hseeberger in here... I believe these are his names.

Reply:
Okay. Have updated to reference the `cloudstate.proxy.spanner` settings, like for project, instance, database. Removed the redundant settings.

Reply:
I'm not sure about the potentially trailing underscore. Let's get this merged quickly to fix the issue at hand and then deal with that underscore later.